src/js/jsx/tools/GuidesOverlay.jsx

```diff
@@ -172,6 +172,10 @@ define(function (require, exports, module) {
          * Draws the guide edge areas
          */
         drawGuideEdges: function () {
+            if (this.state.document.unsupported) {
```

Comment:
Instead of this, can you make a change in Scrim.jsx where the overlays are `null` when the document is unsupported?